CHANGELOG.md

```diff
@@ -1,5 +1,110 @@
 # Changelog
 
+## 0.8.4
+
+### Docker Image
+
+If you have been running a node using the v0.8.3 Docker image, the node was not saving data to the 
+specified [data directory](https://docs.pantheon.pegasys.tech/en/latest/Getting-Started/Run-Docker-Image/#data-directory),
+or referring to the custom [configuration file](https://docs.pantheon.pegasys.tech/en/latest/Getting-Started/Run-Docker-Image/#custom-configuration-file)
+or [genesis file](https://docs.pantheon.pegasys.tech/en/latest/Getting-Started/Run-Docker-Image/#custom-genesis-file). 
+
+To recover the node key and data directory from the Docker container:
+`docker cp <container>:/opt/pantheon/key <destination_file>`
+`docker cp <container>:/opt/pantheon/database <destination_directory>` 
+
+Where `container` is the name or ID of the Docker container containing the Pantheon node. 
+
+The container can be running or stopped when you copy the key and data directory. If your node was 
+fully synchronized to MainNet, the data directory will be ~2TB.  
+
+When restarting your node with the v0.8.4 Docker image:
+
+* Save the node key in the [`key` file](https://docs.pantheon.pegasys.tech/en/latest/Configuring-Pantheon/Node-Keys/#node-private-key) in the data 
+    directory or specify the location using the [`--node-private-key` option](https://docs.pantheon.pegasys.tech/en/latest/Configuring-Pantheon/Node-Keys/#specifying-a-custom-node-private-key-file).  
+* Specify the `<destination_directory` as a [volume for the data directory](https://docs.pantheon.pegasys.tech/en/latest/Getting-Started/Run-Docker-Image/#data-directory). 
+
+### Upcoming Changes in v0.9
+
+#### Breaking Changes to Command Line 
+
+In v0.9, changes will be made to the command line options to improve usability. These will be breaking changes; that is, 
+in many cases the v0.8 command line options will no longer work. The 
+[Pantheon documentation](https://docs.pantheon.pegasys.tech/en/latest/Reference/Pantheon-CLI-Syntax/) will be 
+updated to reflect these changes. Any further information required about the changes will be included in the v0.9 release notes. 
+
+#### Private Network Quickstart 
+
+In v0.9, the Private Network Quickstart will be moved to a separate repository and the existing Private Network 
+Quickstart removed from the Pantheon repository. The [Private Network Quickstart tutorial](https://docs.pantheon.pegasys.tech/en/latest/Getting-Started/Private-Network-Quickstart/) 
+will be updated and use the Private Network Quickstart in the separate repository.
+
+### Bug Fixes
+- Fixing default resource locations inside docker [\#529](https://github.com/PegaSysEng/pantheon/pull/529)
+- NewRoundMessageValidator ignores Round Number when comparing blocks [\#523](https://github.com/PegaSysEng/pantheon/pull/523) 
+- Fix Array Configurable command line options [\#514](https://github.com/PegaSysEng/pantheon/pull/514) 
+
+## Additions and Improvements
```

Comment:
What's the real difference between "improvements" and "technical improvements" ? Maybe either having only one "improvement" title or specifying the type of improvements on "Additions and Improvements" title.

Reply:
Yes - I did wonder about this while I was working on them. Also, the line between the two is a little blurry.  Unless you have a huge objection I think I'll add this comment to the wider release notes tasks I've got - at the moment, it's the same as for the other releases.